src/openloco/graphics/gfx.cpp

```diff
@@ -184,10 +184,82 @@ namespace openloco::gfx
      */
     uint16_t get_string_width(const char* buffer)
     {
-        registers regs;
-        regs.esi = (uintptr_t)buffer;
-        call(0x495685, regs);
-        return regs.cx;
+        uint16_t width = 0;
+        uint8_t* str = (uint8_t*)buffer;
```

Comment:
prefer to use a reinterpret cast over a C style cast. Also this is const data. Don't cast away const.

Reply:
I removed const from the function header.